src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -30,6 +30,9 @@
   for the same information. Note that `server.address` is only included when
   the `EnableConnectionLevelAttributes` option is enabled.
   ([#2229](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2229))
+* Fix bug where `server.port` attribute was captured as a string. It should be an
+  integer.
```

Comment:
nit:

* When `EnableConnectionLevelAttributes` is enabled the `server.port` attribute will now be written as an integer to be compliant with the [semantic conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/server.md). Previously it was written as a string.